filestack/src/main/java/com/filestack/android/internal/CloudListViewHolder.java

```diff
@@ -1,11 +1,12 @@
-package com.filestack.android;
+package com.filestack.android.internal;
 
 import android.content.Context;
 import android.support.v7.widget.RecyclerView;
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.filestack.android.R;
```

Comment:
@shawnmaten What does this do?

Reply:
It's for Android's resource system. The various resources (text, images, vectors, etc) we have under filestack/src/main/res are processed and static references / ids are put into R. So I can do something like getString(R.string.title_text).

https://developer.android.com/guide/topics/resources/accessing-resources.html